package.json

```diff
@@ -66,15 +85,37 @@
     "@types/sinon": "~4.3.3",
     "@types/tapable": "0.2.5",
     "@types/webpack": "3.8.1",
+    "babel-cli": "6.26.0",
+    "babel-core": "6.26.0",
+    "babel-eslint": "8.2.1",
+    "babel-loader": "7.1.2",
+    "babel-plugin-transform-class-properties": "6.24.1",
+    "babel-plugin-transform-object-rest-spread": "6.26.0",
+    "babel-plugin-transform-react-jsx": "6.24.1",
+    "babel-preset-env": "1.6.1",
+    "chai": "4.1.2",
+    "chai-subset": "1.6.0",
+    "classnames": "2.2.5",
     "codecov": "~3.0.4",
     "cpx": "~1.5.0",
+    "css-loader": "0.28.9",
+    "d3": "3.5.17",
+    "del": "3.0.0",
+    "eslint": "4.16.0",
```

Comment:
do we need these if we've removed eslint?

Reply:
Nope. Removed